### ISSUE TITLE
No sound (нет звука)

### ISSUE THREAD

**Issue by sash11** [beeline09/hackintoshSequoiaMs01#1]
I cannot get ALC269 to work on my MS-01. Driver does not load.
У меня на работает взук - драйвер не подгружается для кодека ALC269.

![PCI](https://github.com/user-attachments/assets/9f582294-6f6d-47c8-af5e-765d0eac355a)

**Comment by beeline09** [comment]
> I cannot get ALC269 to work on my MS-01. Driver does not load. У меня на работает взук - драйвер не подгружается для кодека ALC269.
> 
> ![PCI](https://private-user-images.githubusercontent.com/16323875/397897492-9f582294-6f6d-47c8-af5e-765d0eac355a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3NjA5MzAsIm5iZiI6MTczNDc2MDYzMCwicGF0aCI6Ii8xNjMyMzg3NS8zOTc4OTc0OTItOWY1ODIyOTQtNmY2ZC00N2M4LWFmNWUtNzY1ZDBlYWMzNTVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIxVDA1NTcxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzYWVhMzZhZGRkMDE3NmE4MTdmZDc3MTVlNTMxZDMwOGFmM2E0NzhkMDdmM2Y3YzI1Y2Q1ZDUyZTE1NzZiM2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dinJC6RlSs7UW2njgbF_uJjFaMdCtaMFF9A1vnnWQOs)

Да, у меня тоже не работает. Я пользуюсь звуком через HDMI, поэтому не стал заморачиваться...
